roles/base/tasks/main.yml

```diff
@@ -38,13 +38,18 @@
 - name: install and start ntp
   shell: systemctl enable ntpd
   when: ansible_os_family == "RedHat"
+- name: check if golang is installed
+  stat: path=/usr/local/go/bin/go
+  register: golang_bin
 - name: download Golang v1.5.2
   get_url:
```

Comment:
I think we can use the `force=no` attribute of the `get_url` module to prevent a download if the file already exists.